index.html

```diff
@@ -1,21 +1,16 @@
 <!DOCTYPE html>
 <head>
-  <title>TOP POT DONUT DEMAND SIMULATOR</title>
+  <link href="css/style.css" type="text/css" rel="stylesheet" />
+  <link href='http://fonts.googleapis.com/css?family=Fjalla+One|Indie+Flower' rel='stylesheet' type='text/css'>
```

Comment:
Typically the ordering of tags within the head goes something like this: <meta>, <title>, <link>, <script>. You will also want to pay attention to the ordering of your CSS links; recall that they're "Cascading...", and can overwrite each other.